docroot/themes/custom/yukonca_glider/templates/block/block--footer-site-branding.html.twig

```diff
@@ -0,0 +1,30 @@
+{#
```

Comment:
This twig template is for the block in the Footer - Logo region.  I also have code in the .theme file that sets the footer logo image and exposes it to the twig file as the 'site_footer_logo' variable.  I've cleaned up the code now though as I see it was rendering a separate anchor tag than the one already rendered in the pattern template.

Reply:
Footer is already ready on DEV env.  https://dev-yukon-drupal-10.pantheonsite.io/
In case you are trying to fix the issue with empty content on your local, skip it please.
We have few places  with the same issue that this should be fixed. We already have a solution to use 'Fixed Block Content', but there is some incompatibility with D10 should be fixed first.

Reply:
Sure I can remove this block,  but can I suggest we keep it instead of using fixed block content, especially considering there are compatibility issues with that module.  

This is the same block we use on other sites we develop when the footer site logo is different from the header logo because it uses the existing site branding block type with just a small customization to the template and preprocess_block theme hook.  This way the footer logo image is set directly from the code base without having to be uploaded to the site as a media entity and then added to block content. 

I'll remove the 'footer_site_branding' block and restore the 'whitelogo' block.  But how about I leave the template and preprocess_block code in case the fixed block content module does not work out.

Reply:
@willsstrong yep, that sounds good. Please remove the block, we can use later as a backup plan ;)